tracer/src/Datadog.Trace/Headers/IHeaderNormalizer.cs

```diff
@@ -0,0 +1,30 @@
+// <copyright file="IHeaderNormalizer.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.Headers
+{
+    internal interface IHeaderNormalizer
+    {
+        /// <summary>
+        /// Datadog tag requirements:
```

Comment:
Just to make it clear that we're talking about tag names, not tag values.
```suggestion
        /// Datadog tag name requirements:
```

Reply:
Also consider renaming the method parameter from `string value` to `string name`.